homeassistant/components/synology_dsm/const.py

```diff
@@ -27,7 +27,8 @@
 # Configuration
 CONF_VOLUMES = "volumes"
 
-DEFAULT_SSL = True
+DEFAULT_USE_SSL = True
+DEFAULT_VERIFY_SSL = False
```

Comment:
Is it a device limitation that makes us want to not verify ssl by default? Normally in general the correct setting is to verify ssl when using ssl.

Reply:
By default, DSM generates a self signed certificate.

But it can handle multiple certificates, including ones that are signed by trusted authorities.

Actually, before this commit, the integration never validate SSL connections, see https://github.com/ProtoThis/python-synology/issues/48

Reply:
@MartinHjelmare  is this comment solved?

Reply:
Or maybe we should set `DEFAULT_VERIFY_SSL = True` with a breaking change ?

Reply:
I'd expect most users don't have a valid ssl cert on their box.

Reply:
Yes, it's fine since for this device this is the default case.